Cargo.toml

```diff
@@ -11,6 +11,7 @@ members = [
   "micro_rpc_build",
   "micro_rpc_tests",
   "oak_channel",
+  "oak_enclave_rt",
```

Comment:
Does `rt` stand for `runtime`?
I think it may be better to remove abbreviations from crate names (so that a crate name would easily identify what's inside it) ?

Reply:
I renamed it to `oak_enclave_runtime_support`.